packages/astro/src/compiler/transform/doctype.ts

```diff
@@ -9,7 +9,7 @@ export default function (_opts: { filename: string; fileID: string }): Transform
       html: {
         Element: {
           enter(node, parent, _key, index) {
-            if (node.name === '!doctype') {
+            if (node.name === '!doctype' || node.name === '!DOCTYPE') {
```

Comment:
Just checked the spec. It looks like mixed-case `!doctype` is valid as well, so this should probably check for `node.name.toLowerCase() === '!doctype'`

https://html.spec.whatwg.org/multipage/parsing.html#doctype-name-state

Reply:
Ah, yeah I thought of that and figured it would be faster to avoid the conversion (on every tag). I've never seen anyone mix case this. But yeah if that's what the spec says we should do it.

Reply:
Fixed now, please rereview.